plugins/Monolog/Processor/ExceptionToTextProcessor.php

```diff
@@ -76,6 +76,10 @@ private function getMessage($exception)
 
     private function getStackTrace($exception)
     {
+        if (!\Piwik_ShouldPrintBackTraceWithMessage()) {
```

Comment:
@diosmosis can we also check all other places where we use `getTraceAsString()`?